paging-runtime-composeui/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinMultiplatform
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+
+plugins {
+  alias(libs.plugins.kotlin.multiplatform)
+  alias(libs.plugins.mavenPublish)
+  alias(libs.plugins.jetbrains.compose)
+  id("com.android.library")
+  alias(libs.plugins.kotlin.parcelize)
+}
+
+kotlin {
+  android()
+  jvm {
+    compilations.all {
+      kotlinOptions.jvmTarget = "11"
+    }
+  }
+
+  sourceSets {
+    all {
+      languageSettings {
+        optIn("androidx.paging.ExperimentalPagingApi")
+      }
+    }
+    val commonMain by getting {
+      dependencies {
+        api(projects.pagingCommon)
+        implementation(libs.androidx.paging.compose)
```

Comment:
This dependency shouldn't exist as it's incompatible with iOS (and if it was compatible with iOS we wouldn't need this new artifact in the first place 😄)

Reply:
Once https://github.com/cashapp/multiplatform-paging/pull/9#discussion_r1066088984 is done, this _should_ be as easy as deleting the dependency and being done with it. Lmk if it ends up being more involved